spec/views/lectures/index.html.erb_spec.rb

```diff
@@ -24,4 +24,26 @@
     assert_select "tr>td", text: "Enrollment Key".to_s, count: 2
     assert_select "tr>td", text: "created".to_s, count: 2
   end
+
+  it "renders Start button for created lectures" do
+    render
+    assert_select "[value='Start']", count: 2
+  end
+
+  it "renders View and End button for running lectures" do
```

Comment:
Here you set the status directly. Shouldn't be also checked that, the button click changes the status? Or is this already checked by an other test I just didn't see?
- [ ] Check if this behavior is already tested

Reply:
View tests should not test behaviour, only the html output. It's also impossible to click buttons in html spec tests.
The Tests for Ending a Lecture where implemented in #38 where this feature is coming from.